src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -12,43 +12,166 @@
+void func_80B87FDC(EnKaizoku* this, PlayState* play2);
+void func_80B88CD8(EnKaizoku* this);
+void func_80B87E28(EnKaizoku* this);
+void func_80B86B58(EnKaizoku* this);
+void func_80B8971C(EnKaizoku* this, PlayState* play);
+void func_80B894C0(EnKaizoku* this, PlayState* play);
+void func_80B89280(EnKaizoku* this, PlayState* play);
+void func_80B88D6C(EnKaizoku* this, PlayState* play);
+void func_80B88964(EnKaizoku* this, PlayState* play);
+void func_80B887AC(EnKaizoku* this, PlayState* play);
+void func_80B88278(EnKaizoku* this, PlayState* play);
+void func_80B88378(EnKaizoku* this, PlayState* play);
 
 #if 0
-// static DamageTable sDamageTable = {
-static DamageTable D_80B8AB3C = {
-    /* Deku Nut       */ DMG_ENTRY(0, 0x1),
-    /* Deku Stick     */ DMG_ENTRY(1, 0xF),
-    /* Horse trample  */ DMG_ENTRY(0, 0x0),
-    /* Explosives     */ DMG_ENTRY(1, 0xF),
-    /* Zora boomerang */ DMG_ENTRY(0, 0x1),
-    /* Normal arrow   */ DMG_ENTRY(1, 0xF),
-    /* UNK_DMG_0x06   */ DMG_ENTRY(0, 0x0),
-    /* Hookshot       */ DMG_ENTRY(0, 0x1),
-    /* Goron punch    */ DMG_ENTRY(1, 0xE),
-    /* Sword          */ DMG_ENTRY(1, 0xE),
-    /* Goron pound    */ DMG_ENTRY(0, 0xD),
-    /* Fire arrow     */ DMG_ENTRY(2, 0x2),
-    /* Ice arrow      */ DMG_ENTRY(2, 0x3),
-    /* Light arrow    */ DMG_ENTRY(2, 0x4),
-    /* Goron spikes   */ DMG_ENTRY(1, 0xE),
-    /* Deku spin      */ DMG_ENTRY(0, 0x1),
-    /* Deku bubble    */ DMG_ENTRY(1, 0xE),
-    /* Deku launch    */ DMG_ENTRY(2, 0xE),
-    /* UNK_DMG_0x12   */ DMG_ENTRY(0, 0x1),
-    /* Zora barrier   */ DMG_ENTRY(0, 0x5),
-    /* Normal shield  */ DMG_ENTRY(0, 0x0),
-    /* Light ray      */ DMG_ENTRY(0, 0x0),
-    /* Thrown object  */ DMG_ENTRY(1, 0xE),
-    /* Zora punch     */ DMG_ENTRY(1, 0xE),
-    /* Spin attack    */ DMG_ENTRY(1, 0xD),
-    /* Sword beam     */ DMG_ENTRY(0, 0x0),
-    /* Normal Roll    */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x1B   */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x1C   */ DMG_ENTRY(0, 0x0),
-    /* Unblockable    */ DMG_ENTRY(0, 0x0),
-    /* UNK_DMG_0x1E   */ DMG_ENTRY(0, 0x0),
-    /* Powder Keg     */ DMG_ENTRY(1, 0xE),
+typedef enum {
+    /*  0 */ KAIZOKU_WAIT, // TODO: check
+    /*  1 */ KAIZOKU_DEFEAT, // TODO: check
+    /*  2 */ KAIZOKU_DAMAGED, // TODO: check
+    /*  3 */ KAIZOKU_JUMP, // TODO: check
+    /*  4 */ KAIZOKU_ROLL_BACK, // TODO: check
+    /*  5 */ KAIZOKU_READY, // TODO: check
+    /*  6 */ KAIZOKU_BLOCK, // TODO: check
+    /*  7 */ KAIZOKU_SLASH, // TODO: check
+    /*  8 */ KAIZOKU_ADVANCE, // TODO: check
+    /*  9 */ KAIZOKU_PIVOT, // TODO: check
+    /* 10 */ KAIZOKU_CIRCLE, // TODO: check
+    /* 11 */ KAIZOKU_UNUSED, // TODO: check
+    /* 12 */ KAIZOKU_SPIN_ATTACK, // TODO: check
+    /* 13 */ KAIZOKU_SIDESTEP, // TODO: check
+    /* 14 */ KAIZOKU_ROLL_FORWARD, // TODO: check
+    /* 15 */ KAIZOKU_STUNNED, // TODO: check
+    /* 16 */ KAIZOKU_SPIN_DODGE // TODO: check
+} EnKaizokuAction;
```

Comment:
I'd note that this came from OoT, which is why it needs checking.

Reply:
oh lol, I forgot to remove it :sweat_smile: